homeassistant/components/braviatv/config_flow.py

```diff
@@ -177,6 +179,59 @@ async def async_step_confirm(
 
         return self.async_show_form(step_id="confirm")
 
+    async def async_step_reauth(self, entry_data: Mapping[str, Any]) -> FlowResult:
+        """Handle configuration by re-auth."""
+        self.entry = self.hass.config_entries.async_get_entry(self.context["entry_id"])
+        self.device_config = {**entry_data}
+        return await self.async_step_reauth_confirm()
+
+    async def async_step_reauth_confirm(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Dialog that informs the user that reauth is required."""
+        errors: dict[str, str] = {}
```

Comment:
This seems unused and can be removed here and below

Reply:
yeah, this dict is unnecessary, removed.